diff --git a/third-party/zstd/CMakeLists.txt b/third-party/zstd/CMakeLists.txt
index b8c5c35..645f3ce 100644
--- a/third-party/zstd/CMakeLists.txt
+++ b/third-party/zstd/CMakeLists.txt
@@ -1,11 +1,8 @@
 if(LITO_USE_SYSTEM_ZSTD)
-  find_package(zstd CONFIG REQUIRED)
-  if(NOT TARGET zstd::libzstd)
-    message(FATAL_ERROR "The system zstd package does not provide zstd::libzstd")
-  endif()
-
+  find_package(PkgConfig REQUIRED)
+  pkg_check_modules(ZSTD REQUIRED IMPORTED_TARGET libzstd)
   add_library(lito.zstd INTERFACE)
-  target_link_libraries(lito.zstd INTERFACE zstd::libzstd)
+  target_link_libraries(lito.zstd INTERFACE PkgConfig::ZSTD)
   return()
 endif()
 
