--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ build: build-$(OS_NAME)
 	
 .PHONY: build-linux build-darwin
 build-linux:
-	$(CC) -O3 -g sgdboop.c curl-helper.c gui-helper-gtk.c string-helpers.c crc.c $(shell $(PKG_CONFIG) --cflags --libs gtk+-3.0 libcurl) -o SGDBoop $(LDFLAGS)
+	$(CC) $(CFLAGS) sgdboop.c curl-helper.c gui-helper-gtk.c string-helpers.c crc.c $(shell $(PKG_CONFIG) --cflags --libs gtk+-3.0 libcurl) -o SGDBoop $(LDFLAGS)
 	chmod +x SGDBoop
 
 build-darwin:
